intranet/apps/preferences/fields.py

```diff
@@ -25,7 +22,5 @@ def to_python(self, value):
         return value
 
     def widget_attrs(self, widget):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/tjcsl/ion/file/2354287857/issues/source?bid=3176160&fileBranchId=3218556#l24)